data/seeds/001-incidents.js

```diff
@@ -1,61 +0,0 @@
-exports.seed = function (knex) {
```

Comment:
Are we sure we don't want to keep some kind dummy data in the codebase?

Reply:
I'm open to it, so long as whenever we seed the legacy data it is seeded using the specific-file flag